src/utils/conversions.ts

```diff
@@ -28,6 +31,23 @@ export const getServicePayloadUpdater = (user: IExtendedUserContract) => (
   const serviceMetadata: ServiceMetadata = {
     ...payload.service_metadata,
     // Scope for visible services cannot be changed
+    category:
+      (SpecialServiceMetadata.is(payload.service_metadata) ||
```

Comment:
I'd extract these conditions for clarity (ie. `const canSetCustomFlow =` ...)

Reply:
We can reduce code duplication with a method that checks if the category field exists. Conditions are different for the field `category` and `custom_special_flow`. 18855bf
Add some comments in 18855bf